src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/SQLFunctionsTest.java

```diff
@@ -257,7 +257,7 @@ public void functionLogs() throws Exception {
     @Test
     public void functionPow() throws Exception {
         String query = "SELECT pow(account_number, 2) as key,"+
-                "abs(age - 60) as new_age from " + TEST_INDEX_ACCOUNT + "/account limit 1";
+                "abs(age - 60) as new_age from " + TEST_INDEX_ACCOUNT + "/account WHERE firstname = 'Virginia' and lastname='Ayala' limit 1";
```

Comment:
What is this change for?

Reply:
With the same query, the return hit in ES 7.0 is different with ES 6.7. So I add firstname and lastname to make sure the same hit will be returned.
In case, the test is checking the function of pow. it should be fine.